topics/remote/index.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases: [remote-work, remote-collaboration]
+created_by: "@rendred Tech Headhunter"
+display_name: "Remote"
+github_url: "https://github.com/topics/remote"
+logo: "https://www.wontokmail.com/wp-content/uploads/2020/11/1200-x-627-1.jpg"
+related: [remote-work, collaboration, distributed-teams, work-from-home, WFA, WFH, work-life balance]
+short_description: "Explore repositories and resources for effective remote work, collaboration, and managing distributed teams."
+topic: remote
+url: 
+  - "https://en.wikipedia.org/wiki/Remote_work"
+  - "https://github.blog/news-insights/the-library/remote-work-how-global-teams-collaborate-across-time-zones/"
+  - "https://buffer.com/state-of-remote-work/"
+  - "https://trello.com/remote-work-guide"
+  - "https://zapier.com/learn/remote-work/"
+  - "https://asana.com/resources/remote-work"
+  - "https://remoteok.io/"
+  - "https://nomadlist.com/"
+  - "https://remote.tools/"
+  - "https://about.gitlab.com/company/culture/all-remote/guide/"
+  - "https://lisereiner.substack.com/"
```

Comment:
Unless I'm mistaken, `url` is actually not an array property but expects a string 🤔